pages/auth/setup.vue

```diff
@@ -234,7 +238,7 @@ export default {
           </div>
         </template>
 
-        <template v-else>
+        <template v-else-if="mcmEnabled">
```

Comment:
This can go back to v-else - see next comment

Reply:
You need to change the logic for `finishPassword` - it sets the step to 2, regardless of whether mcm is enabled or not, so with this extra v-else-if, you hit continue to set the password and then you end up on an empty step 2 with no where to go.

I changed the line `this.step = 2;` to:

```
        if (this.mcmEnabled) {
          this.step = 2;
        } else {
          this.done();
        }
```